include/numerics/eigen_core_support.h

```diff
@@ -45,7 +45,7 @@
 #include "libmesh/ignore_warnings.h"
 
 // Eigen includes
-#include <Eigen/Dense>
```

Comment:
This, on the other hand, I'm not so sure about.  radial_basis_interpolation.C uses Eigen dense solvers, which aren't in Core, are they?